src/main/java/edu/kis/powp/jobs2d/command/transformers/RotateStrategy.java

```diff
@@ -18,23 +19,27 @@ public RotateStrategy(int x, int y, double angle) {
         }
 
         public OperateToCommand execute(OperateToCommand operateToCommand) {
-            double cos = Math.cos(Math.toRadians(angle));
-            double sin = Math.sin(Math.toRadians(angle));
+            Integer[] newCoordinates = setNewCoordinates(operateToCommand);
 
-            int finalX = (int) (operateToCommand.getPosX() * cos - operateToCommand.getPosY() * sin);
-            int finalY = (int) (operateToCommand.getPosX() * sin + operateToCommand.getPosY() * cos);
-
-            return new OperateToCommand(finalX, finalY);
+            return new OperateToCommand(newCoordinates[0], newCoordinates[1]);
         }
 
         public SetPositionCommand execute(SetPositionCommand setPositionCommand) {
+            Integer[] newCoordinates = setNewCoordinates(setPositionCommand);
+
+            return new SetPositionCommand(newCoordinates[0], newCoordinates[1]);
```

Comment:
now code duplication is between FlipStrategy and RotateStrategy. Now it is possibly to introduce abstract class (template method)